okio/src/jsMain/kotlin/okio/fs.kt

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This class declares the subset of Node.js filesystem APIs that we need in Okio.
```

Comment:
As long as all of these aren't API we may retire them later if Dukat or kotlinx-nodejs improve

Reply:
definitely

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50341/files !